packages/api/src/schema/schema.yaml

```diff
@@ -2567,90 +2518,148 @@ components:
 
 paths:
```

Comment:
As chatted on Discord, we are both missing a lot of relevant APIs here as well as listing APIs that are not relevant to users. I can help come up with the exact list of APIs that should be documented here once we get there.

Reply:
Not exactly sure, but i didn't find any other asset APIs in the schema: https://github.com/livepeer/studio/blob/d6c4db23b0965495b0f637699c8022098122e00f/packages/api/src/schema/schema.yaml#L3198-L3218

There is only a `get`, is the others are on separate file?

Reply:
I think the biggest confusion is that you are assuming this schema is the source of truth! It is not for the `paths`, the only really used section of this schema has been the `components` section!

As for the APIs, the source of truth are the API controllers code. They are all in the `src/controllers` folder and you can generally find the APIs easily there, declared with express routers. The schemas here are either incomplete sometimes, or totally hallucinating hahaha. In case of assets/VOD, it is missing a bunch of APIs like request-upload etc. In case of users, it's listing APIs that don't really exist, like that other comment I made. 

Just to be clear, this is a tech debt issue on this current schema, and not a problem with this PR. But part of the work in this API reference project is cleaning up that tech debt and documenting here the actual APIs that we want in our API reference. This is the context-dependent part of the work that I mentioned to you I can help on the low-level. Let's just sort the bigger picture first (like how to separate the API&DB schemas).